### PR TITLE
6090 improve user status context menu

### PR DESCRIPTION
**Pull request by endulab**
### What does the PR do

Adjust UserStatusContextMenu to Figma design:
https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?node-id=77%3A19108

Requires StatusQ: https://github.com/status-im/StatusQ/pull/749

### Affected areas

UserStatusContextMenu

### Screenshot of functionality

image

### CODE REVIEW

**Comment by endulab** [comment]
The pr contains commit 8d6dbdf which is a part of #6058.
You can review only last commit.

**Comment by osmaczko** [comment]
Changes are duplicated with https://github.com/status-im/status-desktop/pull/6058

Could you please make `5886-adding-automatic-status-logic` the merge target? or mark it as a draft until https://github.com/status-im/status-desktop/pull/6058 is merged to master?

**Review by elina2015 — APPROVED** [review]
Tested and Approved

**Comment by endulab** [comment]
> Changes are duplicated with #6058
> 
> Could you please make `5886-adding-automatic-status-logic` the merge target? or mark it as a draft until #6058 is merged to master?

@osmaczko #6058 was merged. I rebased the branch.

**Review by osmaczko — APPROVED** [review]
:+1:

File: ui/imports/shared/popups/UserStatusContextMenu.qml [hunk 1]
```diff
@@ -28,74 +25,55 @@ PopupMenu {
...
-        icon.source: Style.svg("online")
-        icon.width: 16
-        icon.height: 16
     }
 
-    Action {
+    StatusMenuItem {
+        id: inactiveAction
         text: qsTr("Inactive")
+        image.source: Style.svg("statuses/inactive")
```
Comment (PascalPrecht) [inline ui/imports/shared/popups/UserStatusContextMenu.qml:59]: We **can** do it this way, but we could also introduce these as icons and have something like `icon.name:"status-inactive"`

Just spitballing here :)

**Review by PascalPrecht — APPROVED** [review]

**Pull request merged**